### DISCUSSION TITLE
Using `SELECT EXISTS`

### DISCUSSION THREAD

**Discussion by lukewhrit** [jackc/pgx#1427] [Q&A] [answered]
How should I go about executing a statement that looks like this: `SELECT EXISTS(SELECT 1 FROM profile WHERE email = $1)`; using the `EXISTS` operator. I just want the default `t` or `f` returned but pgx always returns something else, like a "No rows returned" error

**Comment by jackc** [comment, marked as answer]
Works for me:

```go
package main

import (
	"context"
	"log"
	"os"

	"github.com/jackc/pgx/v5"
)

func main() {
	ctx := context.Background()

	conn, err := pgx.Connect(ctx, os.Getenv("DATABASE_URL"))
	if err != nil {
		log.Fatal(err)
	}
	defer conn.Close(ctx)

	var b bool
	err = conn.QueryRow(ctx, "select exists(select 1 from pg_type where typname = 'int4')").Scan(&b)
	if err != nil {
		log.Fatal(err)
	}

	log.Println(b)
}
```

Output:

```
2022/12/15 20:30:35 true
```

**Reply by lukewhrit** [reply]
Yeah I realized earlier I was using Exec instead of QueryRow and that that isn't what you're supposed to do.

Thanks
